### DISCUSSION TITLE
Keepass TOTP import

### DISCUSSION THREAD

**Discussion by bingoxo** [ente-io/ente#1636] [Enhancements]
### Description

hi , i have 2fa stored in keepass and trying to export it to ente auth but no luck , tried unencrypted csv html xml , none of them work , can you take a look at it please ? , i exported those formats from encrypted kdbx database and decrypted them with keepassdx

### Version

2.0.54

### What product are you using?

Ente Auth

### What platform are you using?

Desktop - Windows

**Comment by KaKi87** [comment]
Hi @vishnukvmd,
Any news on this ?
Thanks :pray:
